org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizard.java

```diff
@@ -56,55 +60,84 @@ public boolean performFinish() {
 		}
 
 		if (!template.isEmpty()) {
```

Comment:
The `dotnet new` command does not do anything when no template is provided, only lists available templates.